schema.sql

```diff
@@ -13,3 +13,26 @@ CREATE TABLE IF NOT EXISTS public.animals
 
 ALTER TABLE animals ADD species VARCHAR(50)
 
+/* Create owners table */
+CREATE TABLE owners(
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(100),
+    age INT
+);
+
+/*Create Species table */
+CREATE TABLE species(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(50)
+);
+
+/*Constraints */
```

Comment:
- [ ] There is one more to add : 
- Remove column `species` by
 `ALTER TABLE REMOVE species;`